File: MatricesJava/src/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
    public static void main(String[] args) {
   Scanner scanner = new Scanner(System.in);

        System.out.print("Por favor ingrese el numero de filas: ");
        int cantidadFilas = scanner.nextInt();
        System.out.print("Por favor ingrese el numero de columnas: ");
        int cantidadColumnas = scanner.nextInt();
        int sumaColumnas[] = new int[cantidadColumnas];
        int sumaFilas[] = new int[cantidadFilas];
        int suma;
        /*
        3. Hacer un programa que llene una matriz de 7 * 7. Calcular la suma de cada renglón y almacenarla en un vector, la suma de cada columna y almacenarla en otro vector.
        pero cambia 7 *7 po lo que la persona ingrese el imite
         */
        int matriz[][] = new int[cantidadFilas][cantidadColumnas];

        for (int i = 0; i < cantidadFilas; i++) {
            suma = 0;
            for (int j = 0; j < cantidadColumnas; j++) {
                System.out.print("Por favor ingrese un numero: ");
                matriz[i][j] = scanner.nextInt();
                suma += matriz[i][j];
            }
            sumaFilas[i] = suma;
        }

        for (int i = 0, aux = 1; i < cantidadColumnas; i++, aux++) {
            suma = 0;
                for (int j = 0; j < cantidadFilas; j++) {
                    suma += matriz[j][i];
            }
            sumaColumnas[i] = suma;
        }

        for (int i = 0; i < cantidadFilas; i++) {
            System.out.println("Suma fila #" + (i + 1) + ": " + sumaFilas[i]);
        }

        for (int i = 0; i < cantidadColumnas; i++) {
            System.out.println("Suma columna #" + (i + 1) + ": " + sumaColumnas[i]);
        }
    }
}

File: MatricesC++/Ejercicio3.cpp
#include<iostream>
#include<conio.h>

using namespace std;

int main(){
	/*Hacer un programa que llene una matriz de 7*7. Calcular la suma de cada renglón y almacenarla en un vector, la suma de cada columna y almacenarla en otro vector*/
	int matriz[10][10];
	int cantidadFilas  = 0, cantidadColumnas = 0, sumasFilas[cantidadFilas], sumasColumnas[cantidadColumnas];
	int sumaFila = 0, sumaColumna = 0; 
	cout<<"Por favor ingrese la cantiadd de filas: "; cin >> cantidadFilas; 
	cout<<"Por favor ingrese la cantidad de columnas: "; cin >> cantidadColumnas;
	for(int i = 0; i < cantidadFilas; i++){
		sumaFila = 0;
		for(int j = 0; j < cantidadColumnas; j++){
			cout<<"Por favor ingrese un numero para la posicion: ("<<i+1<<"-"<<j+1<<"): "; cin >> matriz[i][j];
			sumaFila += matriz[i][j];
		}
		sumasFilas[i] = sumaFila; 
	}	
	for(int i = 0; i < cantidadColumnas; i++){
		sumaColumna = 0;
		for(int j = 0; j < cantidadFilas; j++){
			sumaColumna += matriz[j][i];
		}
		sumasColumnas[i] = sumaColumna;
	}
	for(int i = 0; i < cantidadFilas; i++){
		cout<<"La suma de la fila : "<<i+1<<" es: "<<sumasFilas[i]<<endl;
	}
	
	for(int i = 0; i < cantidadColumnas; i++){
		cout<<"La suma de la Columna : "<<i+1<<" es: "<<sumasColumnas[i]<<endl;
	}
	getch();
	return 0; 
}
